### ISSUE TITLE
Seekbar

### ISSUE THREAD

**Issue by Selenali24** [Reivax2003/TexasHoldemCS301#2]
When choosing how much to fold/bet, if I move the seek bar all the way to the left so that the value is $0, nothing happens at all. So the game only works for values greater than but not including $0 even though the seek bar goes to 0. Maybe adjust the starting value of the seek bar so that the player will not be able to go to 0 (for whatever reason that they may).

**Comment by cheyanneey** [comment]
For the first run of the game, it showed up on the emulator correctly but was buggier on the tablet. What we found was that the game on the first run of the tablet would not have a correctly working dumb AI or that AI would not do what it was intended to do. While it was supposed to match the bet of the player, it would actually raise the bet and we were able to see that through the value in the pot. This was a concern as I was aware that the dumb AI was only programmed to match the bet. During this first run I also had the understanding that new cards would show up after each bet and all I had seen were my starting cards after betting
